Makefile

```diff
@@ -49,6 +49,7 @@ verify-scripts:
 	bash -x hack/verify-integration-tests.sh
 	bash -x hack/verify-group-versions.sh
 	bash -x hack/verify-prerelease-lifecycle-gen.sh
+	hack/verify-payload-crds.sh
```

Comment:
why doesn't this one need a `bash -x` whatever that may be?

Reply:
`bash -x` sets the debug flag, meaning that it will print out every single detail about the script when it executes. Personally I don't think we need this at the moment. Our scripts seem to be fairly reliable and the output from each one should be sculpted to give sufficient detail, that debug logs aren't necessary. 

For example on this script, it prints which file it diffs before the diff kills the script, and if it finds additional files, it prints the name of the additional file and instructs the user how to correct the issue